unexpected change of local (and global) variables from within another function (Python 3.8.12) [duplicate]/questions/72869951/unexpected-change-of-local-and-global-variables-from-within-another-function72869951
